tests/frontend/test_circuitspecs_chip2.py

```diff
@@ -305,15 +376,16 @@ def test_no_unitary(self, tol):
         assert program_equivalence(res, expected, atol=tol)
 
     def test_interferometers(self, tol):
-        """Test interferometers correctly decompose to MZ gates"""
+        """Test that the compilation correctly decomposes the interferometer using
+        the rectangular_symmetric mesh (which decomposes to MZ gates)"""
```

Comment:
just remove the mention of MZ gates, since it doesn't seem to be relevant for the actual test

Reply:
:+1: